src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --font-family: 'Roboto', Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could use 'sans-serif' as a generic fallback.